src/Microsoft.Health.Fhir.Core/Features/Definition/SearchParameterDefinitionManager.cs

```diff
@@ -261,15 +263,33 @@ private async Task LoadSearchParamsFromDataStore(CancellationToken cancellationT
                         catch (SearchParameterNotSupportedException ex)
                         {
                             _logger.LogWarning(ex, "Error loading search parameter {Url} from data store.", searchParam.GetStringScalar("url"));
+                            foreach (OperationOutcomeIssue issue in ex.Issues)
+                            {
+                                issueDetails.Append(issue.Diagnostics);
+                                issueDetails.AppendLine();
+                            }
                         }
                         catch (InvalidDefinitionException ex)
```

Comment:
Maybe we can simplify this part of the code with:

`
catch (FhirException ex)
{
    StringBuilder issueDetails = new StringBuilder();
    foreach (OperationOutcomeIssue issue in ex.Issues)
    {
        issueDetails.Append(issue.Diagnostics).Append(";");
    }
    
    _logger.LogWarning(ex, "Error loading search parameter {Url} from data store. Issues: {Issues}", searchParam.GetStringScalar("url"), issueDetails);
}
`

Reply:
As InvalidDefinitionException  and SearchParameterNotSupportedException  inherit from FhirException, a single catch block can simplify the code.